MekHQ/src/mekhq/gui/stratcon/ScenarioWizardLanceModel.java

```diff
@@ -26,12 +30,24 @@
  * @author NickAragua
  */
 public class ScenarioWizardLanceModel extends DefaultListModel<Force> {
+    private static final long serialVersionUID = 5015606735749150646L;
+
     /**
      * Constructor - sometimes, you have a list of force IDs.
      */
     public ScenarioWizardLanceModel(Campaign campaign, Collection<Integer> forceIDs) {
+        List<Force> sortedForces = new ArrayList<>(); 
+        
         for (int forceID : forceIDs) {
-            super.addElement(campaign.getForce(forceID));
+            sortedForces.add(campaign.getForce(forceID));    
+        }
+        
+        // let's sort these guys by alphabetical order
+        Collections.sort(sortedForces, 
+                (Comparator<Force>) (Force o1, Force o2) -> o1.getName().compareTo(o2.getName()) );
```

Comment:
I'd agree, but the compared items are technically Force objects rather than strings, and I'd rather keep it as a Model<Force> so I can just retrieve the selected force rather than muck around with IDs.

Reply:
```
        final NaturalOrderComparator noc = new NaturalOrderComparator();
        sortedForces.sort((f1, f2) -> noc.compare(f1.getName(), f2.getName()));
```

Does the sort properly and uses the human order sort.

However, I think a SortedComboBoxModel using the above force comparator would actually be the best implementation.

Reply:
And... that also gives me and idea. I could add an override of DefaultComboBoxModel::getSelectedItem that returns null or E, instead of Object...